bsky-components/help/HelpPreview.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+const emit = defineEmits<{
+  (event: 'close'): void
+}>()
+
+const vAutoFocus = (el: HTMLElement) => el.focus()
+</script>
+
+<template>
+  <div my-8 px-3 sm:px-8 md:max-w-200 flex="~ col gap-4" relative>
+    <button v-auto-focus type="button" btn-action-icon absolute top--8 right-0 m1 :aria-label="$t('action.close')" @click="emit('close')">
+      <span i-ri:close-line />
+    </button>
+
+    <img :alt="$t('app_logo')" :src="`/${''}logo.svg`" w-20 h-20 height="80" width="80" mxa class="rtl-flip">
+    <h1 mxa text-4xl mb4>
+      {{ $t('help.title') }}
+    </h1>
+    <p>
+      {{ $t('help.desc_para1') }}
+    </p>
+    <p>
+      <b text-primary>{{ $t('help.desc_highlight') }}</b>
+      {{ $t('help.desc_para2') }}
+    </p>
+    <p>
+      {{ $t('help.desc_para4') }}
+      <NuxtLink font-bold text-primary href="https://github.com/elk-zone/elk" target="_blank">
+        {{ $t('help.desc_para5') }}
+      </NuxtLink>
+      {{ $t('help.desc_para6') }}
+    </p>
+    <NuxtLink hover:text-primary href="https://github.com/sponsors/elk-zone" target="_blank">
```

Comment:
Maybe not very useful now, but I guess even if there is no sponsors link for Nimbus yet at least it's already ready to go.

```suggestion
    <NuxtLink hover:text-primary href="https://github.com/sponsors/nimbus-town" target="_blank">
```